redfish-core/lib/log_services.hpp

```diff
@@ -5122,7 +5125,7 @@ static bool fillPostCodeEntry(
         bmcLogEntry["Name"] = "POST Code Log Entry";
         bmcLogEntry["Id"] = postcodeEntryID;
         bmcLogEntry["Message"] = std::move(msg);
-        bmcLogEntry["MessageId"] = "OpenBMC.0.2.BIOSPOSTCode";
+        bmcLogEntry["MessageId"] = "OpenBMC.0.4.BIOSPOSTCodeASCII";
```

Comment:
This is the correct version, upstream we bump this version when we add a new message. Here this seems reasonable we just stay with .4